apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -163,6 +167,25 @@ sort_map_list_field(Field, Map) ->
         _ -> Map
     end.
 
+enable_autocluster() ->
+    ok = ekka:enable_autocluster(),
+    _ = ekka:autocluster(emqx),
+    emqx_ctl:print("Automatic cluster discovery enabled.~n").
+
+maybe_disable_autocluster() ->
+    case ekka:autocluster_enabled() of
+        true ->
+            ok = ekka:disable_autocluster(),
+            emqx_ctl:print(
+                "Automatic cluster discovery is disabled on this node: ~p to avoid"
+                " re-joing the same cluster again, if the node is not stopped soon."
```

Comment:
```suggestion
                " re-joining the same cluster again, if the node is not stopped soon."
```

Reply:
:facepalm: Thanks a lot, forgot to run a spell check again... Fixed.